src/controllers/userController.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const userModel = require('../models/userModel');
+
+function createUser(req, res) {
+  const { name, email } = req.body;
+
+  if (!name && !email) {
+    return res.status(400).send('Name or email are required');
+  }
+
+  const newUser = userModel.createUser(name, email);
+
+  res.status(201).json(newUser);
+}
+
+function getAllUsers(req, res) {
+  const users = userModel.getAllUsers();
+
+  res.json(users);
+}
+
+function getUserById(req, res) {
+  const user = userModel.getUserById(parseInt(req.params.id));
```

Comment:
You could use a similar solution to what you did for Expenses, where _getAllUsers_ could accept a query and solve all cases at once.
Then you would be able to search users not only by Id, but by any wanted prop